src/lib/api/pr_info.ts

```diff
@@ -58,6 +58,38 @@ export async function getPrInfoForBranches(
   }
 
   const octokit = new Octokit({ auth });
+
+  const existingNewPrRequests = [];
+  for (const ref of branchesWithoutPrInfo.values()) {
+    existingNewPrRequests.push({
+      pr: octokit.request(
+        'GET /repos/{owner}/{repo}/pulls?head={owner}:{branch_name}',
```

Comment:
This do not works well with fork (like this PR). Owner of the repo is not the same than user of the ref.
Here it is `bradymadden97/freephite` and `ligro:add-already-existing-prs`.